index.html

```diff
@@ -837,7 +837,14 @@ <h2><dfn>Implicit Pointer Capture</dfn></h2>
         </section>
         <section>
             <h3>Implicit Release of Pointer Capture</h3>
-            <p>Immediately after firing the <code>pointerup</code> or <code>pointercancel</code> events, a user agent MUST clear the <a>pending pointer capture target override</a> for the <code>pointerId</code> of the <code>pointerup</code> or <code>pointercancel</code> event that was just dispatched, and then run <a href="#process-pending-pointer-capture">Process Pending Pointer Capture</a> steps to fire <code>lostpointercapture</code> if necessary.</p>
+            <p>
```

Comment:
a minor thing, but maybe worth not leaving `<p>` and `</p>` hanging on their own lines